src/ui/units/dash/store/actions/dashTyped.ts

```diff
@@ -656,22 +647,9 @@ export const applyControl2Dialog = () => {
 };
 
 export const applyGroupControlDialog = () => {
```

Comment:
Let's move all new actions for group selectors to separate folder/file inside dash/store/actions. And the same thing for reducers and selectors. 
I've started example with new relations.
We have issue to refactor long dash store files, so let's add new ones in new way.